### PR TITLE
Resolve Issue #107: As a map user, I want the Address text box to update, so that I know where I am searching.

### PR DESCRIPTION
**Pull request by drothschild**
Hosted on https://github.com/drothschild/sf-crime-data

### CODE REVIEW

File: CNAME [hunk 1]
```diff
@@ -1 +0,0 @@
-sfcrimedata.org
```
Comment (squanto) [inline CNAME:1]: for sure we want to keep this guy

**Comment by paulhennings** [comment]
Hi David: This is the "live test" site link I was looking for: https://drothschild.github.io/sf-crime-data --Paul

File: CNAME (hunk 1 above)
Comment (paulhennings) [reply CNAME:1]: Hugo, Yes, This is the only thing keeping me from hitting the "PULL" button right now.

However, I'm not clear that the presence of the CNAME file in the root is still required for Github Pages hosting anymore. Github Help says to set the custom domain in Settings, and does not specify that CNAME is required (see Ref: below). There is still Github Help for troubleshooting if your CNAME file is wrongly configured, but no "first setup" instruction for CNAME. I can't find a clear statement from Github about whether CNAME is required, or is deprecated in favor of using Settings.

That said, the CNAME file is a PITA. It's only valid on the "live" site. It causes an error to be thrown for developers testing the site on their personal Github Pages, (like David's hosting at https://drothschild.github.io/sf-crime-data).

Thoughts? Does anyone have clearer info from Github?

If not, I may "test in production" some evening. =:o If the pull "as is" works without CNAME, great. If it causes the site to die, CNAME can easily be re-added.

Ref: https://help.github.com/articles/adding-or-removing-a-custom-domain-for-your-github-pages-site/

**Comment by paulhennings** [comment]
And the result of the "test in production" is that deleting the CNAME file made sfcrimedata.org throw a 404. I put CNAME back in the root and the live site was happy. CNAME is indeed still required.

**Pull request merged**